nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -46,12 +47,36 @@ public class Serializer {
   /** Path to write suggested fix metadata. */
   private final Path fieldInitializationOutputPath;
 
+  /**
+   * Version for all serialized outputs. Outputs formats may change overtime, this version is
+   * serialized to keep track of changes.
+   */
+  private static final int SERIALIZATION_VERSION = 1;
+
   public Serializer(FixSerializationConfig config) {
     String outputDirectory = config.outputDirectory;
     this.errorOutputPath = Paths.get(outputDirectory, "errors.tsv");
     this.suggestedFixesOutputPath = Paths.get(outputDirectory, "fixes.tsv");
     this.fieldInitializationOutputPath = Paths.get(outputDirectory, "field_init.tsv");
     initializeOutputFiles(config);
+    serializeVersion(outputDirectory);
+  }
+
+  /**
+   * Serializes the {@link Serializer#SERIALIZATION_VERSION} as {@code string} in
+   * <b>serialization_version.txt</b> file under root output directory for all serialized outputs.
+   *
+   * @param outputDirectory Path to root directory for all serialized outputs.
+   */
+  private void serializeVersion(String outputDirectory) {
+    Path versionOutputPath = Paths.get(outputDirectory).resolve("serialization_version.txt");
+    try (Writer fileWriter =
+        Files.newBufferedWriter(versionOutputPath.toFile().toPath(), Charset.defaultCharset())) {
+      fileWriter.write(SERIALIZATION_VERSION + "");
+      fileWriter.flush();
```

Comment:
I don't think we need this `flush()` call, since the writer gets closed immediately (which implies a flush)?

Reply:
IntelliJ was only making the call to `close` gray not the `flush`, but removed it anyway. I think it is not necessary. cce08362b5e2b17d08315258e607395e6e1b9200.